checkov/arm/checks/resource/ACRGeoreplicated.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+from typing import Any
+from checkov.common.models.consts import ANY_VALUE
+from checkov.common.models.enums import CheckCategories
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+
+
+class ACRGeoreplicated(BaseResourceValueCheck):
+    def __init__(self) -> None:
+
+        name = "Ensure geo-replicated container registries to match multi-region container deployments."
+        id = "CKV_AZURE_165"
+        supported_resources = ("Microsoft.ContainerRegistry/registries/replications",)
```

Comment:
@SaraWeinberg1234 @ChanochShayner I believe this needs to be a graph check that ensures `Microsoft.ContainerRegistry/registries` resources are connected to `Microsoft.ContainerRegistry/registries/replications`. location is a [required](https://learn.microsoft.com/en-us/azure/templates/microsoft.containerregistry/registries/replications?pivots=deployment-language-arm-template) field, so I believe this check will always pass (the fail UT is not valid).

Reply:
If this should be a graph check let's close this PR, I don't think we currently support edges in ARM.